### ISSUE TITLE
Icons missing

### ISSUE THREAD

**Issue by modingly** [closebox73/Scorpio#1]
![image](https://user-images.githubusercontent.com/93032925/171090677-c737819f-66e9-4599-a240-dfb823465128.png)

I am missing the icons for memory, weather, home, cpu etc...

My city is not supported by the map.. Is there any other way to add my exact location?

**Comment by modingly** [comment]
Yes i did.. Problem solved after using this command Sudo apt install
conky-all jq curl

But still weather icon is missing 🤔

**Comment by vishnutganesh** [comment]
There is a symbol A cap before the Degree Celsius. Can anyone fix that too

**Comment by closebox73** [comment]
@vishnutganesh Have all the fonts been installed and updated the cache?

**Comment by modingly** [comment]
> @modingly try this, copy weathericons-regular-webfont.ttf into .fonts folder in your ~ (home), open terminal and execute fc-cache -fv

yes i did. but it is not solved...

**Comment by modingly** [comment]
> There is a symbol A cap before the Degree Celsius. Can anyone fix that too

I have this issue too..

**Comment by Gauthamsudo** [comment]
I am also have the following issues. I am using Graffias theme and I am also stuck with it. If there is any update, Please report.

**Comment by closebox73** [comment]
@modingly if you using antares theme, try to change Carlito to Roboto
![Screenshot_20220605-190146.png](https://user-images.githubusercontent.com/91961267/172049564-a338e0b9-2892-4b53-a8e4-e2fa8d856e90.png)

**Comment by Gauthamsudo** [comment]
Does anybody have Solution for Graffias theme?
I've been trying.

**Comment by vishnutganesh** [comment]
Tried installing font. But didn't work. The problem is degree symbol. When I removed it. A cap is no longer appeared.

**Comment by closebox73** [comment]
@vishnutganesh @modingly 
so the problem is with the degree symbol, maybe this can help, try taking the degree symbol from another source (eg a weather forecast website or other) then replace the one in antares.conf / graffias.conf.

**Comment by ghost03x** [comment]
> 

can u show us how to do that??

**Comment by vishnutganesh** [comment]
![Screenshot from 2022-06-11 00-04-07](https://user-images.githubusercontent.com/66023554/173129594-d0e05eb2-8516-49eb-b440-cc29ff76737e.png)

I added degree symbol from website to .config file. This happened.

**Comment by vishnutganesh** [comment]
Code :

`
conky.text = [[
${offset 0}${voffset 10}${font Metropolis Black:size=12}${time %A  %d %B %Y}${font Cantarell:size=1.5}
${offset 0}${voffset 15}${color}${font League Spartan:bold:size=35}${execi 100 cat ~/.cache/weather.json | jq '.main.feels_like' | awk '{print int($1+0.8)'}} °C. ${execi 100 cat ~/.cache/weather.json | jq -r '.weather[0].description' | sed -e 's/\(.*\)/\U\1/'}${font Cantarell:size=1}
${execi 300 ~/.config/conky/Graffias/scripts/weather.sh}\
${offset 0}${voffset 25}${color1}${font Carlito:size=11}Local time in ${execi 100 cat ~/.cache/weather.json | jq -r '.name'}  is ${time %H:%M} Hrs. Today Max temperature is
${offset 0}${voffset 4}${color1}${font Carlito:size=11}${execi 100 cat ~/.cache/weather.json | jq '.main.temp_max' | awk '{print int($1+0.5)'}} °C. with wind speed ${execi 100 (cat ~/.cache/weather.json | jq '.wind.speed')}km/h and ${execi 100 (cat ~/.cache/weather.json | jq '.main.humidity')}% humidity
${offset 41}${voffset 70}${font Metropolis Black:size=9}CPU >> ${cpu cpu0}%${goto 196}RAM >> ${memperc}%`

**Comment by closebox73** [comment]
@vishnutganesh
try to change font type in Graffias.conf ( line 70 ) from Spartan league to roboto
then save.
`${font League Spartan:bold:size=35}` to `${font Roboto:bold:size=35}`

**Comment by Gauthamsudo** [comment]
Don't understand a thing, Sir. If you have any simpler method, Then please comment.

**Comment by closebox73** [comment]
@Gauthamsudo what do you not understand?

**Comment by closebox73** [comment]
@Gauthamsudo ghost03x solving his problem by moving to Arch OS..

**Comment by Gauthamsudo** [comment]
I am using Ubuntu 22.04

**Comment by Gauthamsudo** [comment]
I really downloaded and used this from LinuxScoop - A youtubers pack of theme.

**Comment by Gauthamsudo** [comment]
Hello, I tried @vishnutganesh Code only but It was not showing. Is there any fix? This is my default code :

**Comment by Gauthamsudo** [comment]
conky.text = [[

${offset 0}${voffset 10}${font Metropolis Black:size=12}${time %A. %d %B %Y}${font Cantarell:size=1}

${offset 0}${voffset 15}${color}${font League Spartan:bold:size=35}${execi 100 cat ~/.cache/weather.json | jq '.main.temp' | awk '{>

${execi 300 ~/.config/conky/Graffias/scripts/weather.sh}\

${offset 0}${voffset 25}${color1}${font Carlito:size=11}Local time in ${execi 100 cat ~/.cache/weather.json | jq -r '.name'}  is ${>

${offset 0}${voffset 4}${color1}${font Carlito:size=11}${execi 100 cat ~/.cache/weather.json | jq '.main.temp_max' | awk '{print in>

${offset 41}${voffset 70}${font Metropolis Black:size=9}CPU >> ${cpu cpu0}%${goto 196}RAM >> ${memperc}%

${offset 0}${voffset 40}${font Material:size=9}${if_running mocp} ${font Carlito:size=11}${moc_state} :${else}${font 
Material:size>

${offset 0}${voffset 5}${font Metropolis black:size=15}${if_running mocp}${moc_artist}${else}${font}

${offset 0}${voffset 2}${font Carlito:size=11}${if_running mocp}${moc_song}${else}

${image  ~/.config/conky/Graffias/res/oval.png -s 132x40 -p 0,200}

${image ~/.config/conky/Graffias/res/ring.png -s 132x40 -p 150,200}

]]

**Comment by Gauthamsudo** [comment]
@closebox73 This is the same config I use.

**Comment by closebox73** [comment]
> conky.text = [[
> 
> ${offset 0}${voffset 10}${font Metropolis Black:size=12}${time %A. %d %B %Y}${font Cantarell:size=1}
> 
> ${offset 0}${voffset 15}${color}${font League Spartan:bold:size=35}${execi 100 cat ~/.cache/weather.json | jq '.main.temp' | awk '{>
> 
> ${execi 300 ~/.config/conky/Graffias/scripts/weather.sh}\
> 
> ${offset 0}${voffset 25}${color1}${font Carlito:size=11}Local time in ${execi 100 cat ~/.cache/weather.json | jq -r '.name'}  is ${>
> 
> ${offset 0}${voffset 4}${color1}${font Carlito:size=11}${execi 100 cat ~/.cache/weather.json | jq '.main.temp_max' | awk '{print in>
> 
> ${offset 41}${voffset 70}${font Metropolis Black:size=9}CPU >> ${cpu cpu0}%${goto 196}RAM >> ${memperc}%
> 
> ${offset 0}${voffset 40}${font Material:size=9}${if_running mocp} ${font Carlito:size=11}${moc_state} :${else}${font 
> Material:size>
> 
> ${offset 0}${voffset 5}${font Metropolis black:size=15}${if_running mocp}${moc_artist}${else}${font}
> 
> ${offset 0}${voffset 2}${font Carlito:size=11}${if_running mocp}${moc_song}${else}
> 
> ${image  ~/.config/conky/Graffias/res/oval.png -s 132x40 -p 0,200}
> 
> ${image ~/.config/conky/Graffias/res/ring.png -s 132x40 -p 150,200}
> 
> ]]
> 

a lot is missing from this code.. just use the one from my repository, if it's still the same maybe try using another theme.. I have made dozens of them

**Comment by Gauthamsudo** [comment]
@closebox73 I have only put conky.text file only. There is the other conky.config[[ thing. I will use your repository Sir. Actually this conky theme - I found it from a YouTuber called LinuxScoop. I downloaded his package and used it.

**Comment by closebox73** [comment]
@Gauthamsudo yeah,he is my friend

**Comment by Gauthamsudo** [comment]
Oh, It was his code.

**Comment by Gauthamsudo** [comment]
@closebox73 I used your theme but now It is in the bottom in a black background

**Comment by closebox73** [comment]
@Gauthamsudo never mind bro, every error and problem will give me a new knowledge
maybe i'll late to answer because i'm in work or another busy..
keep trying to fix your error,, its fun right??

**Comment by Gauthamsudo** [comment]
Yes bro, You are right. Ok then.
